addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: defaultstorageclass-protection
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.0.5-2"
-    appversion.kubeaddons.mesosphere.io/defaultstorageclass-protection: "0.0.5"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.0-0"
```

Comment:
i'm wondering why 1.1.0 and not 1.0.0 but okay, works for me!

Reply:
No real reason.. Sort of it's not new so I felt like 1.0.0 was wrong.